### DISCUSSION TITLE
ART 1.19.0

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2540] [Announcements]
This release of ART 1.19.0 introduces Steal Now Attack Later and Rescaling Auto Conjugate Gradient attacks and the Be Your Own Neighbourhood Detector (BEYOND) for adversarial examples.

# Added

- Added the Steal Now Attack Later (SNAL) evasion attack (#2440)
- Added the Rescaling Auto Conjugate Gradient (ReACG) descent evasion attack (#2460)
- Added the Be Your Own Neighbourhood Detector (BEYOND) for adversarial examples in PyTorch (#2489)
- Added support for scikit-learn models with multiples outputs (#2505)

# Changed

- Changed  AutoAttack to allow defining number of processes used in parallel processing (#2529)

# Removed

[None] 

# Fixed

- Fixed use of deprecated function `binom_test` from `scipy` (#2517)
- Fixed bug in random sampling of patch locations in masks for adversarial patch attacks in PyTorch (#2539)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.19.0'>ART 1.19.0</a>.</em>

**Comment by dirtyoldman65** [comment]
Hello, 
I'm exploring universal adversarial patches that cause image classifiers to label inputs as “dog,” regardless of the original content. I’m particularly interested in using a **pre-generated patch image** if available, because I don't have access to a GPU or local environment to run patch training.

I found your project [PROJECT NAME] and thought you might already have a patch targeting the “dog” class, or a sample image generated in your experiments. Would you be willing to share it or point me to where I might download one?

I'm using it solely for personal exploration around privacy. I'll adhere to any licensing requirements or usage terms you specify.

Thank you for your work and time.

Best regards,  
[Terry]
